src/util/helpers.js

```diff
@@ -102,10 +77,6 @@ export const legendFields = [
     '!userGroupAccesses',
 ]
 
-export const legendSetFields = [
```

Comment:
This was not used?

Reply:
When I grep the whole code base, this is the only instance of it.